### PR TITLE
round11-lion-beta1-0p8: Lion β1=0.8 counter-experiment to PR #159

### PR DESCRIPTION
**Pull request by morganmcg1**
## Hypothesis

Lion β1 controls the EMA-of-gradient term used inside the sign-update:
`update = sign(β1 · m + (1 − β1) · g)`. The current SOTA (PR #115, thorfinn) uses Lion's default β1=0.9. PR #159 (fern) tested **β1=0.95** (more smoothing, only 5% fresh gradient contribution) and regressed across every metric — abupt_mean 11.151 vs SOTA 10.580 (+5.4%), with the gap *growing* through epochs and the worst regression on tau_y (+1.106 abs).

This PR runs the **symmetric counter-experiment: β1=0.8** (less smoothing, **20%** fresh-gradient contribution vs 10% at β1=0.9, vs 5% at β1=0.95).

**Mechanistic prediction**: PR #159 showed the SOTA stack benefits from *less* smoothing, not more. β1=0.8 should:
1. Track high-frequency boundary-gradient signal (tau_y, tau_z) more aggressively — these were the most-degraded metrics at β1=0.95.
2. Speed up early convergence — PR #115 ep1 abupt was 53.75; expect lower at β1=0.8 if the signal/noise tradeoff is on the "more responsive" side.
3. Possibly hurt stability if too aggressive — Lion already operates on signs, so noise tolerance is generally high; β1=0.8 is well-precedented in the original Lion paper for smaller batches.

This is the highest-information-value follow-up: it is a single delta vs SOTA, exactly mirrors the tested direction (β1=0.95) on the opposite side, and directly tests whether the SOTA stack wants *less* gradient smoothing.

## Baseline (PR #115, thorfinn)

- W&B run: `d03oghpp` (project: `wandb-applied-ai-team/senpai-v1-drivaerml-ddp8`)
- **Primary**: `test_primary/abupt_axis_mean_rel_l2_pct = 10.580` (SOTA)
- Best `val_primary/abupt_axis_mean_rel_l2_pct = 9.484` at epoch 9
- Stack: Lion lr=1e-4 wd=5e-4 EMA=0.999 cosine-T_max=50 layers=4 hidden=512 heads=8 slices=128 no-compile DDP8 batch=4 epochs=9
- Default β1=0.9, β2=0.99

## Experiment

**Single delta**: `--lion-beta1 0.8` (everything else exactly matches SOTA).

```bash
torchrun --standalone --nproc_per_node=8 train.py \
  --optimizer lion --lr 1e-4 --weight-decay 5e-4 --batch-size 4 --epochs 9 \
  --use-ema --ema-decay 0.999 --lr-cosine-t-max 50 \
  --model-layers 4 --model-hidden-dim 512 --model-heads 8 --model-slices 128 \
  --train-surface-points 65536 --train-volume-points 65536 \
  --eval-surface-points 65536 --eval-volume-points 65536 \
  --no-compile-model \
  --lion-beta1 0.8 \
  --wandb_group round11-lion-beta1-0p8
```

Verify the flag spelling with `python train.py --help | grep -i beta` before launching — adjust if the actual flag name differs (e.g. `--lion_beta1`, `--optimizer-beta1`).

## What to watch / report

Please report in the PR comments:
1. **Primary**: `test_primary/abupt_axis_mean_rel_l2_pct` (vs SOTA 10.580). Anything < 10.580 is a merge candidate.
2. **Best val abupt** + epoch (vs SOTA 9.484 @ ep9).
3. **Per-axis test metrics**: `tau_x`, `tau_y`, `tau_z`, `cp` `rel_l2_pct` (vs SOTA — paying special attention to tau_y, which was the worst axis under β1=0.95).
4. **Convergence curve**: val abupt at ep1, ep3, ep5, ep7, ep9. Compare against PR #115 (ep1=53.75, ep9=9.484) and PR #159 (β1=0.95, gap grew through training).
5. **Volume metrics** if reported (volume pressure rel_l2_pct).
6. W&B run URL.

## Decision criteria

- **Beats SOTA on test abupt_mean** → merge candidate.
- **Better than PR #115 on val but not on test** → leave a comment, advisor will decide.
- **Worse than SOTA but ep1/ep3 show clear faster convergence** → potentially worth combining with another change (e.g. shorter T_max). Note the signal in your comment.
- **Strict regression on every metric** like PR #159 → close as dead-end with the data; this would establish that β1=0.9 is at the optimum for the SOTA stack on this dataset.

## Constraints

- Single delta. No other hyperparameters change vs SOTA.
- Do not change the model architecture, dataset, or DDP setup.
- Do not change `SENPAI_MAX_EPOCHS` or `SENPAI_TIMEOUT_MINUTES`.
- 8×H100 DDP, ~270 min budget, 9 epochs.

### CODE REVIEW

**Comment by morganmcg1** [comment]
ADVISOR: Note on W&B group flag — the reproduce command in the PR body uses `--wandb_group` (underscore), but the correct flag is `--wandb-group` (hyphen). If your training is already running, this won't affect the model results, but the W&B run may appear in a different group than expected.

When you report results, please include the actual W&B run URL so I can locate the run regardless of which group it landed in. No action needed mid-run — just note this for your final report.

**Comment by morganmcg1** [comment]
ADVISOR: Closing PR #189 because All 24 training runs (3 batches of 8 GPUs) failed with zero metrics — no validation data to evaluate. The reproduce command in the PR body uses `--wandb_group` (underscore) but the correct flag is `--wandb-group` (hyphen); this may have caused the failures. Additionally, a comment was left on 2026-05-01 about this flag issue but the runs could not be recovered.

The hypothesis (Lion β1=0.8 counter-experiment vs PR #159's β1=0.95 regression) remains scientifically interesting. However, given repeated infrastructure failures and the availability of better-performing directions (edward's lr-cosine-tmax-9 is already at 10.61% val), closing this to free fern's GPU for a fresh, higher-priority experiment.

If you want to revisit Lion β1 tuning, use `--wandb-group tay-round13-lion-beta1-sweep` and confirm flags carefully against `python train.py --help` before submitting..

**Comment by morganmcg1** [comment]
ADVISOR/Codex correction: this PR did produce usable W&B metrics; it was a completed negative result, not a zero-metric infrastructure failure.

Rank-0 W&B run `iukbkkgm` (`fern/round11-lion-beta1-0p8-rank0`) finished with:
- `best_val_primary/abupt_axis_mean_rel_l2_pct`: 10.0929
- `full_val_primary/abupt_axis_mean_rel_l2_pct`: 10.0929
- `test_primary/abupt_axis_mean_rel_l2_pct`: 11.2425
- `test_primary/surface_pressure_rel_l2_pct`: 6.1199
- `test_primary/wall_shear_rel_l2_pct`: 11.1437
- `test_primary/volume_pressure_rel_l2_pct`: 13.1461

Compared with current SOTA PR #115 / run `d03oghpp` (`test_abupt=10.5801`), this is not a winner, so keeping the PR closed is fine. I updated the label from `status:wip` to `status:closed` to avoid future polling/status ambiguity.
